test/config_parser_SUITE_data/mongooseim-pgsql.cfg

```diff
@@ -25,10 +25,11 @@
                                          {timeout, infinity},
                                          {ping_rate, none},
                                          {max_stanza_size, 100}
-                                                                         ]}
-                                      ]},
-                            {"localhost", "/api", mongoose_api_admin, [{auth, {<<"ala">>, <<"makotaipsa">>}}]},
-                            {"localhost", "/api/contacts/{:jid}", mongoose_api_client_contacts, []}
```

Comment:
Why did `mongoose_api_client_contacts` change to `mongoose_api_client`?

Reply:
I believe it was a mistake taken from [this doc](https://mongooseim.readthedocs.io/en/latest/advanced-configuration/Listener-modules/), should have been `mongoose_api_client` in the first place

Reply:
But then why the `path` part still contains `contacts`? 😕

Reply:
Ok, I am documenting this right now, let me sort out the docs with this path as well.